--- Dieterbe-uzbl-4994115/examples/config/config~	2009-12-22 18:07:54.000000000 +0100
+++ Dieterbe-uzbl-4994115/examples/config/config	2010-01-04 08:39:42.000000000 +0100
@@ -331,7 +331,7 @@
 
 # Default context menu
 menu_add Google = set uri = http://google.com
-menu_add Go Home = set uri = http://uzbl.org
+menu_add Go Home = set uri = http://pld-linux.org
 menu_separator separator_1
 menu_add Quit uzbl = exit
 
@@ -370,4 +370,4 @@
 # === Post-load misc commands  ===============================================
 
 # Set the "home" page.
-set uri = uzbl.org/doesitwork/@COMMIT
+set uri = http://pld-linux.org
